sources/MemProtector.cpp

```diff
@@ -40,14 +42,44 @@ std::ostream& operator<<(std::ostream& os, const PLH::ProtFlag flags) {
 		os << "n";
 	else
 		os << "-";
-
-	if (flags & PLH::ProtFlag::P)
-		os << " private";
-	else if (flags & PLH::ProtFlag::S)
-		os << " shared";
 	return os;
 }
 
+PLH::MemoryProtector::MemoryProtector(const uint64_t address, const uint64_t length, const PLH::ProtFlag prot,
+									  bool unsetOnDestroy /*= true */)
```

Comment:
this should be in the body of the constructor. Function calls in the initilizer list are against my style, this is code logic inherent to how the class operates, not simple initialization logic.

Reply:
will revert this